docs/source/model_design/concept_model/ensemble_distributions/index.rst

```diff
@@ -34,8 +34,78 @@ Ensemble Distributions in GBD
 What Is an Ensemble Distribution?
 ---------------------------------
 
+In GBD, many risk factor exposures are modeled by continuous probability
+distributions. Occasionally, if we're lucky, the exposure data may closely
+follow a standard probability distribution, such as normal, lognormal, Weibull,
+gamma, Gumbel, etc. But in general there is no reason to expect real-world data
+to match these nicely parameterized, mathematically convenient models. Thus, in
+order to model exposure data more accurately, GBD uses an `ensemble learning`_
+approach, fitting multiple distributions to the data, and averaging the
+distribution functions together to get an overall "ensemble" distribution that
+fits the data better than any of its individual components.
+
+.. _ensemble learning: https://www.toptal.com/machine-learning/ensemble-methods-machine-learning#:~:text=Ensemble%20methods%20are%20techniques%20that,winning%20solutions%20used%20ensemble%20methods.
+
 Mathematical Definition of Ensemble Distributions
 -------------------------------------------------
 
+What Base Distributions Are Used in GBD's Ensembles?
+----------------------------------------------------
+
+Here are links to R code and Python code implementing GBD's ensemble distributions:
+
+* `R code for ensemble distributions <R code_>`_ on Stash, accessible on the
+  cluster at :file:`/ihme/code/risk/ensemble/` and maintained by Central Comp
+
+* `Python code for ensemble distributions <Python code_>`_ on GitHub, part of
```

Comment:
This looks like a package used generally for distributions, with built in functionality for ensembles? I can't quite tell--but that link is a little opaque to someone unfamiliar with the package---maybe something within the repo would be more illustrative? like https://github.com/ihmeuw/risk_distributions/blob/0f2ac26cf79ca014f502c2b84f6ab8b0af947351/src/risk_distributions/risk_distributions.py#L427

Reply:
It looks to me like **all** of the current contents of the `risk_distributions` module are for implementing GBD's ensemble distributions in Vivarium. I.e. the `BaseDistribution` class and the classes for the named distributions implement the conversion of mean and standard deviation to the appropriate distribution parameters like the R code does, and then the `EnsembleDistribution` class combines the base distributions as a mixture. Is that correct, @KjellSwedin?  Are there any plans to add additional functionality to this module that would be more general than ensemble distribution implementation?

I am inclined to leave the documentation pointing to the entire module since it's all relevant, and request that some additional documentation be added to the module, because it's hard to understand what the different pieces are doing without context...

Reply:
I'll try to reorganize/reword this section to make it more clear what's in the repos.